schemas/2015-02-01-preview/Microsoft.Logic.json

```diff
@@ -47,102 +37,9 @@
         "properties"
       ],
       "description": "Microsoft.Logic/workflows"
-    },
-    "workflows_accessKeys": {
```

Comment:
please remove the references schemas/2014-04-01-preview/deploymentTemplate.json and schemas/2015-01-01/deploymentTemplate.json.